docs/source/frameworks/fastai.rst

```diff
@@ -0,0 +1,246 @@
+   validation**. By putting the :code:`save_model` call to the end of your training
+   pipeline, all your finalized models can be managed in one place and ready for
+   inference.
+
+Optionally, you may attach custom labels, metadata, or :code:`custom_objects` to be
+saved alongside your model in the model store, e.g.:
+
+.. code:: python
+
+    bentoml.fastai.save_model(
+        "fastai_sentiment",       # model name in the local model store
+        learner,                  # model instance being saved
+        labels={                  # user-defined labels for managing models in Yatai
+            "owner": "nlp_team",
+            "stage": "dev",
+        },
+        metadata={                # user-defined additional metadata
+            "dataset_version": "20210820",
+        },
+        custom_objects={          # save additional user-defined python objects
+            "tokenizer": tokenizer,
+        }
+    )
+
+- **labels**: user-defined labels for managing models, e.g. :code:`team=nlp`, :code:`stage=dev`.
+- **metadata**: user-defined metadata for storing model training context information or model evaluation metrics, e.g. dataset version, training parameters, model scores.
+- **custom_objects**: user-defined additional python objects, e.g. a tokenizer instance, preprocessor function, model configuration json, serialized with cloudpickle. Custom objects will be serialized with `cloudpickle <https://github.com/cloudpipe/cloudpickle>`_.
+
+.. note::
+
+   By default, BentoML uses ``cloudpickle`` for learner serialization. The key difference from Python's ``pickle`` is that 
+   ``cloudpickle`` has the capability to serialize functions and so it can directly serialize members of the object without reference to its type.
+
+   If the :func:`save_model` method failed while saving a given learner, your learner may contain a :obj:`~fastai.callback.core.Callback` that is not picklable.
+   All FastAI callbacks are stateful, which makes some of them not picklable. Use :func:`Learner.remove_cbs` to remove unpicklable callbacks.
+
+.. tip::
+
+   We found that some of given callback are only useful during training phase, not during serving. Therefore, removing callback will not affect inference results.
+
+
+Loading a learner with BentoML
+------------------------------
+
+.. warning::
+
+   We recommend users to to use ``load_model`` inside a :obj:`bentoml.Service`.
+
+   You should always use ``bentoml.models.get("model:tag").to_runner()`` to get
+   a :obj:`bentoml.Runner` instead. See also :ref:`Runners <concepts/runner:Using Runners>`_ for more information.
+
+
+To load the learner back to memory, use ``load_model``:
+
+.. code-block:: python
+
+   learner = bentoml.fastai.load_model("fastai_sentiment:latest")
+
+You can then proceed to test the learner with prediction inputs:
+
+.. code-block:: python
+
+   learner.predict("I really liked that movie!")
```

Comment:
This section is conflicting with the message above. I understand you are trying to say that `load_model` can be used in testing. I feel we do not need to include it here as it is not specific to fastai and may cause confusion with the message above.

Reply:
Is it better if I move the warning above to below L173?